docs/commands/changeset-derive-replacement.asciidoc

```diff
@@ -23,10 +23,13 @@ IDs in the secondary data are not retained and those elements treated as complet
 * +input2+                       - Map input with the features to be used for replacement; may be any supported input format that also supports 
                                    bounded reading (e.g. OSM file); leave empty to cut data out of input1 while adding no replacement data. 
                                    See the "Input Data and "Bounds Handling" sections for more detail.
-* +bounds+                       - Boundary over which to do the feature replacement of the form: minx,miny,maxx,maxy; 
-                                   e.g. "-71.4698,42.4866,-71.4657,42.4902" See the ""Bounds Handling" section for more detail.
+* +bounds+                       - Boundary over which to do the feature replacement. Can be either a rectangular bounding box of the form: 
+                                   minx,miny,maxx,maxy; e.g. "-71.4698,42.4866,-71.4657,42.4902" or a closed polygon of the form:
+                                   x1,y1;x2,y2;x3,y3;...; e.g. "-71.4698,42.4866,-71.4657,42.4902". See the ""Bounds Handling" section for 
```

Comment:
Is this meant to be "-71.4698,42.4866**_;_**-71.4657,42.4902" because of  x1,y1;x2,y2;x3,y3;...;?

Reply:
yeah, typo. thanks.